### DISCUSSION TITLE
Stable Video Diffusion Image-to-Video Model from Stability AI

### DISCUSSION THREAD

**Discussion by kubilaykilinc** [lllyasviel/Fooocus#1004] [General]
Amazing new era for ai generated video from Stability AI
check it out

Official article
https://stability.ai/news/stable-video-diffusion-open-ai-video-model

Huggingface models
https://huggingface.co/stabilityai/stable-video-diffusion-img2vid-xt

Civitai
https://civitai.com/models/207992?modelVersionId=234212

**Comment by Charuru** [comment]
We don't need illyasviel to be working on an UI for this, we need him to make controlnet!

**Reply by jepjoo** [reply]
For now I agree. I don't think SVD needs to be rushed into Fooocus at a very early stage.

But once SVD matures a bit I think a user friendly UI would be good for the community to have whether it be Fooocus or something else.

LoRAs should be out soon and that'll already be one additional layer of complexity. The more complex SVD becomes the more awkward ComfyUI becomes for casual users.

I personally also like to use Fooocus to make the initial images for SVD. Having everything in one program would be nice. Now I'm having Fooocus + ComfyUI running simultaneously which isn't an issue for a 4090 but might be for people with less VRAM.

**Reply by Charuru** [reply]
Yeah I would love fooocus for svd but that's not the point, the point is that the developer of svd is the inventor of controlnet! That's a much more interesting technology that other people in the community wouldn't be able to do that he needs to work on.

**Comment by james-smith-saas** [comment]
Hi ,

"Yeah, you can definitely use Fooocus for frame-by-frame animations! 
If you’ve got a full-body image and pivot animations, Fooocus can help overlay the images and make it feel more realistic. You’d just need to use the upscale or vary tabs to tweak the frames and add some smooth transitions. 
If you want more detail, you can even try upscale x2. It’s a bit of a process, but if you’ve got the time, you can really create something cool with it. 

There are also other tools like Runway AI video generator.
 LunaBloom AI does text-to-video generation with prompts within minutes. 
DeepBrain AI also helps with similar tasks for quick videos or more advanced animations. 
"
